drivers/entropy/entropy_nrf5.c

```diff
@@ -10,6 +10,11 @@
 #include <soc.h>
 #include <hal/nrf_rng.h>
 
+#define DT_DRV_COMPAT	nordic_nrf_rng
+
+#define IRQN		DT_INST_IRQN(0)
```

Comment:
Why are we not passing `nordic_nrf_nrg` to the `DT_INST_` macros instead of having to "hardcode it" at the top with the `#define DT_DRV_COMPAT	nordic_nrf_rng` ? I assume this is how it is intended to be done, but I find it a bit too little explicit.

Reply:
The idea of the DT_INST_ macros is that they utilize DT_DRV_COMPAT to reduce having a slew of `DT_INST(0, nordic_nrf_rng)` throughout the driver.  One is still free to use that style if they want or need to.